ogre_media/fonts/ogre1.10.fontdef

```diff
@@ -1,3 +1,12 @@
+Liberation Sans
+{
+  type truetype
+  source liberation-sans/LiberationSans-Regular.ttf
+  size 18
+  resolution 96
+}
+
+# Ogre >= 1.10 has changed the format
```

Comment:
So I think it's actually Ogre > 1.10 that has changed format, not >= 1.10. As we use Ogre 1.10 in RoboStack, this causes issues. But to be confirmed.

Reply:
why did you end up with 1.10? You could solve this by updating to 1.12

Reply:
Does rviz work with 1.12? If yes, we could totally attempt a build!

Reply:
`noetic-devel` does, yes, I tried getting it set as the default for noetic actually but gazebo had already released their last ever release and have some issues with it

Reply:
Yeah we also have gazebo on conda forge and it would be better if rviz and gazebo are side by side installable, so it's the same problem. We could build a variant with updated ogre though